ui/Screen/Org/Projects.svelte

```diff
@@ -2,54 +2,81 @@
   import { push } from "svelte-spa-router";
 
   import * as path from "../../src/path.ts";
-
-  import { Text, Button } from "../../DesignSystem/Primitive";
-  import Placeholder from "../../DesignSystem/Component/Placeholder.svelte";
+  import { Flex, Icon, Text } from "../../DesignSystem/Primitive";
+  import {
+    AdditionalActionsDropdown,
+    List,
+    ProjectCard,
+    Remote,
+    Stats,
+  } from "../../DesignSystem/Component";
+  import { orgs as orgsStore, fetchProjectList } from "../../src/org.ts";
 
   export let params = null;
 
-  const registerProject = () => push(path.registerProject(params.id));
-  const registerMember = () => push(path.memberRegistration(params.id));
+  const select = (event) => {
+    const orgProject = event.detail;
+    if (orgProject.maybeProject) {
+      push(path.projectSource(orgProject.maybeProject.id));
+    }
+  };
+
+  $: fetchProjectList({ id: params.id });
```

Comment:
Would put this at the end of the script tag after all props and consts have been set up.